gddo-server/assets/templates/home.html

```diff
@@ -27,10 +27,10 @@
       <li><a href="/-/index">Index</a>
       <li><a href="/-/go">Go Standard Packages</a>
       <li><a href="/-/subrepo">Go Sub-repository Packages</a>
-      <li><a href="https://code.google.com/p/go-wiki/wiki/Projects">Projects @ go-wiki</a>
-      <li><a href="https://github.com/search?o=desc&q=language%3Ago&s=stars&type=Repositories">Most stars</a>, 
-        <a href="https://github.com/search?o=desc&q=language%3Ago&s=forks&type=Repositories">most forks</a>, 
-        <a href="https://github.com/search?o=desc&q=language%3Ago&s=updated&type=Repositories">recently updated</a> on GitHub
+      <li><a href="https://golang.org/wiki/Projects">Projects @ go-wiki</a>
+      <li><a href="https://github.com/search?o=desc&amp;q=language%3Ago&amp;s=stars&amp;type=Repositories">Most stars</a>, 
```

Comment:
As @garyburd said, it's correct to escape the url inside html, but browsers do accept unescaped.

When I'm feeling lazy and don't want to think hard about escaping rules, I just let the machine do the work, see below. :P

```Go
package main

import (
	"net/url"
	"os"

	"golang.org/x/net/html"
)

func main() {
	url := (&url.URL{
		Scheme: "https",
		Host:   "github.com",
		Path:   "/search",
		RawQuery: url.Values{
			"o":    {"desc"},
			"q":    {"language:go"},
			"s":    {"stars"},
			"type": {"Repositories"},
		}.Encode(),
	}).String()

	a := &html.Node{
		Type: html.ElementNode, Data: "a",
		Attr: []html.Attribute{{Key: "href", Val: url}},
	}

	err := html.Render(os.Stdout, a)
	if err != nil {
		panic(err)
	}
	
	// Output:
	// <a href="https://github.com/search?o=desc&amp;q=language%3Ago&amp;s=stars&amp;type=Repositories"></a>
}
```